distribution/test/itests/test-itests-common/src/main/java/ddf/common/test/catalog/CatalogTestCommons.java

```diff
@@ -0,0 +1,164 @@
+import java.util.Map;
+
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.xml.xpath.XPathExpressionException;
+
+import org.apache.http.HttpStatus;
+
+import com.jayway.restassured.response.Response;
+import com.jayway.restassured.response.ValidatableResponse;
+
+import ddf.common.test.AbstractIntegrationTest;
+
+public class CatalogTestCommons {
+
+    private static final String GEOJSON_NEAR_METACARD = "GeoJson near";
+
+    private static final String GEOJSON_FAR_METACARD = "GeoJson far";
+
+    private static final String PLAINXML_NEAR_METACARD = "PlainXml near";
+
+    private static final String PLAINXML_FAR_METACARD = "PlainXml far";
+
+    private static final String CSW_RESOURCE_ROOT = "/TestSpatial/";
+
+    private static final String CSW_METACARD = "CswRecord.xml";
+
+    public static String ingest(String data, String mimeType) {
+
+        return given().body(data)
+                .header(HttpHeaders.CONTENT_TYPE, mimeType)
+                .expect()
+                .log()
+                .all()
+                .statusCode(HttpStatus.SC_CREATED)
+                .when()
+                .post(REST_PATH.getUrl())
+                .getHeader("id");
+    }
+
+    public static String ingest(String data, String mimeType, boolean checkResponse) {
+        if (checkResponse) {
+            return ingest(data, mimeType);
+        } else {
+            return given().body(data)
+                    .header(HttpHeaders.CONTENT_TYPE, mimeType)
+                    .when()
+                    .post(REST_PATH.getUrl())
+                    .getHeader("id");
+        }
+    }
+
+    public static String ingestGeoJson(String json) {
+        return ingest(json, "application/json");
+    }
+
+    public static Map<String, String> ingestMetacards(Map<String, String> metacardsIds) {
+        //ingest csw
+        String cswRecordId = ingestCswRecord(getFileContent(
+                CSW_RESOURCE_ROOT + "csw/record/CswRecord.xml"));
+        metacardsIds.put(CSW_METACARD, cswRecordId);
+
+        //ingest xml
+        String plainXmlNearId = ingest(getFileContent(
+                CSW_RESOURCE_ROOT + "xml/PlainXmlNear.xml"), MediaType.TEXT_XML);
+        String plainXmlFarId = ingest(getFileContent(
+                CSW_RESOURCE_ROOT + "xml/PlainXmlFar.xml"), MediaType.TEXT_XML);
+        metacardsIds.put(PLAINXML_NEAR_METACARD, plainXmlNearId);
+        metacardsIds.put(PLAINXML_FAR_METACARD, plainXmlFarId);
+
+        //ingest json
+        String geoJsonNearId = ingestGeoJson(getFileContent(
+                CSW_RESOURCE_ROOT + "json/GeoJsonNear.json"));
+        String geoJsonFarId = ingestGeoJson(getFileContent(
+                CSW_RESOURCE_ROOT + "json/GeoJsonFar.json"));
+        metacardsIds.put(GEOJSON_NEAR_METACARD, geoJsonNearId);
+        metacardsIds.put(GEOJSON_FAR_METACARD, geoJsonFarId);
+
+        return metacardsIds;
+    }
+
+    public static String ingestCswRecord(String cswRecord) {
```

Comment:
I've created a follow up ticket to this work that involves refactoring common code out of itest classes. I believe we should determine that once we have a full scope of all the reusable code we have for itests, I'm fine with either route.